src/planning/task_planner/scripts/firebase_node.py

```diff
@@ -9,7 +9,7 @@
 import os
 class FirebaseNode:
     def __init__(self, schema, operation_mode_callback, teleop_mode_callback):
-        firebase_secrets_path = os.path.expanduser("~/ws/api_keys/firebase_secrets.json")
+        firebase_secrets_path = os.path.expanduser("~/firebasesecrets.json")
```

Comment:
lets standardize this and make it ".firebase_secrets.json"

Reply:
I believe the file resides as `.alfred-auxilio-firebase-adminsdk.json` on both our robots